cmds/wifi/wifi.go

```diff
@@ -40,21 +46,23 @@ func main() {
 
 	flag.Parse()
 	a := flag.Args()
-	if len(a) != 2 && len(a) != 1 {
-		flag.Usage()
-		os.Exit(1)
-	}
 	essid = a[0]
 
-	if len(a) == 2 {
+	switch {
```

Comment:
`switch len(a) {`?

Reply:
I almost did that but ... this may get more complex. wifi is just chock full of surprises. So I left it this way.